src/os/bluestore/BlueFS.cc

```diff
@@ -1268,6 +1271,8 @@ int BlueFS::_replay(bool noop, bool to_stdout)
 
   boost::dynamic_bitset<uint64_t> used_blocks[MAX_BDEV];
 
+  set<FileRef> wal_files_to_update; // list of Files to get list of flushes with _wal_update_size 
+
```

Comment:
It would be easier to just iterate over all `file_map` and do `_wal_update_size()` when `f->is_new_wal()`.
Otherwise, handling of OP_FILE_REMOVE is missing.

Reply:
true